_posts/blog/2019-08-15-2019-08-15-lab-meeting.md

```diff
@@ -0,0 +1,128 @@
+(No one was able to help her 😭)
+
+We did have some suggestions around ways to manage her TODO lists though.
+
+* An infinite list: [https://dynalist.io](https://dynalist.io)
+* Trello: [https://trello.com](https://trello.com)
+* Asana: [https://asana.com](https://asana.com/)
+* Pomodoro technique: [https://en.wikipedia.org/wiki/Pomodoro_Technique](https://en.wikipedia.org/wiki/Pomodoro_Technique)
+  * An online timer: [http://e.ggtimer.com/pomodoro](http://e.ggtimer.com/pomodoro)
+* Toggl: [https://toggl.com](https://toggl.com/)
+* Having an *accountabilibuddy* to keep you focused when you could easily be derailed by other tasks.
+* Doing the thing you don't want to do first so there is a reward at the end (and so you don't spend lots of time on it!)
+
+**Maxine** and the [One Health Tech](https://www.onehealthtech.com/) community are [looking for international terms](https://twitter.com/One_HealthTech/status/1161266349013909504?s=20) to monitor ethnic diversity at their events.
+The best solution we found was to use just the top level from the [UK groupings](https://design-system.service.gov.uk/patterns/ethnic-group/) but we did also discuss tools that can summarise free text for the specific purpose that is needed.
+For example the [GenderCoder](https://ropenscilabs.github.io/gendercoder) tool lets people enter their gender in free text so they can self identify, and also facilitates grouping free text answers that are all versions of the same gender (eg: `M`, `male`, `a man`, `Male` are all grouped together.)
+
+This led really well onto a discussion that **Georgia** raised about monitoring diversity for her project.
+Some suggestions were to have continuous monitoring and looking at diversity in leadership.
+In general the take home message was that recording nothing makes reflections almost impossible in the future!
+
+**Yini** asked for the group's opinion on two journals (NeuroImage and Human Brain Mapping) going open access.
+Kirstie had a strong opinion that these are business model changes from the publishers and the open access fees do not make the journals all that more "open" or "accessible".
+In fact, in writing up this blog she found a recent blog about ["Why open access will boost publisher profits"](https://blogs.lse.ac.uk/impactofsocialsciences/2019/06/04/the-gold-rush-why-open-access-will-boost-publisher-profits/) which makes that point really clearly.
+
+**Alex** asked for tips and tricks for reviewing the literature when starting out on a new project.
+We recommended this [10 Simple Rules](https://collections.plos.org/ten-simple-rules) paper by Marco Pautasso in 2013 (doi: [10.1371/journal.pcbi.1003149](https://doi.org/10.1371/journal.pcbi.1003149))
+
+## Rethinking lab meeting
+
+Although the discussion points above filled a lot of lab meeting *[*😻 *-- KW]* we still discussed Kirstie's [proposed guidelines](https://github.com/WhitakerLab/Onboarding/pull/15) for lab meeting.
```

Comment:
I don't know how to view this blog as a webpage, so this might be irrelevant - but the left square bracket is showing with asterisks around instead of being in italics when I view the post as Markdown. Do you need a space before the cat face? Or will it be magically fixed when it's a webpage rather than Markdown?

Reply:
Good catch @LouiseABowler - it does look ok when served with jekyll but thank you for checking!! (the instructions to build locally are linked from the README, but don't worry about that at the moment!)